FiveCalls/FiveCalls/IssueContactDetail.swift

```diff
@@ -142,22 +142,18 @@ struct IssueContactDetail: View {
                     .padding(.bottom)
                 
                 if remainingContacts.count > 1 {
-                    NavigationLink(value: IssueDetailNavModel(issue: issue, contacts: nextContacts)) {
-                        OutcomesView(outcomes: issue.outcomeModels, report: { outcome in
-                            let log = ContactLog(issueId: String(issue.id), contactId: currentContact.id, phone: "", outcome: outcome.status, date: Date(), reported: true)
-                            store.dispatch(action: .ReportOutcome(issue, log, outcome))
-                            store.dispatch(action: .GoToNext(issue, nextContacts))
-                        })
-                    }
+                    OutcomesView(outcomes: issue.outcomeModels, navModel: IssueDetailNavModel(issue: issue, contacts: nextContacts), report: { outcome in
```

Comment:
nit: can we clean up some of the duplication here? seems like the only difference between the two is whether to include `nextContacts` or not and the nav model.